src/main/resources/application.yml

```diff
@@ -21,12 +21,4 @@ logging:
       springframework:
         security: DEBUG
 
-spring:
```

Comment:
DO NOT DELETE THAT - it will rollback to H2 database. Use this only for development

Reply:
sorry, I forgot